packages/ods/src/components/pagination/src/constants/pagination-per-page.ts

```diff
@@ -0,0 +1,20 @@
+enum ODS_PAGINATION_PER_PAGE {
+  min = 10,
```

Comment:
TBD not sur the `min`, `max` value is that great here, would be much more consistent to have same defined value for each, like `option_10`